### ISSUE TITLE
Review: missing dependency on Newtonsoft.Json

### ISSUE THREAD

**Issue by cgravill** [CSBiology/FSharpGephiStreamer#15]
**Describe the bug**
It looks like the NuGet has a missing dependency on Newtonsoft.Json

```
CreateJsonFailed
  "Could not load file or assembly 'Newtonsoft.Json, Version=11.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed'. The system cannot find the file specified."
```

I'm able to workaround this which I did by installing Newtonsoft.Json but you should express the dependency on the NuGet

**OS and framework information (please complete the following information):**
 - OS: Windows 10
 - .Net core SDK version: 2.2.203

https://github.com/openjournals/joss-reviews/issues/1445

**Comment by kMutagene** [comment]
hi @cgravill,

Newtonsoft.Json is listed in the dependencies file:
```
framework: auto-detect

source https://nuget.org/api/v2

nuget FSharp.Core
nuget Newtonsoft.Json //<- here
nuget Argu
...
...
```

Can i ask you to check 2 things? There should be a nuget package downloaded to `src/packages`, whats the version of that file? Second, when you go to `C:\Users\YourUserName\.nuget\packages`, whats the version of the package there?

**Comment by cgravill** [comment]
Ah I wasn't specific, I'm making use of NuGet you're building:
https://www.nuget.org/packages/FSharpGephiStreamer/

![image](https://user-images.githubusercontent.com/1099725/58764996-17904500-8566-11e9-93da-0fe43eee4400.png)

In C:\Users\<MyUserName>\.nuget\packages\Newtonsoft.Json\ I have many versions of Newtonsoft.Json from other projects.

**Comment by cgravill** [comment]
Btw what I suspect is causing the issue is this file:
https://github.com/CSBiology/FSharpGephiStreamer/blob/2c5a3060061b8a33a98a329e193edab629216993/src/FSharpGephiStreamer/paket.template

where you're using a project type of "file" (https://fsprojects.github.io/Paket/template-files.html) so you have to be explicit about everything. I also use that in places, but "project" might work more easily in this instance YMMV.

**Comment by kMutagene** [comment]
If i understand you correctly, the problem shows itself here right?

![image](https://user-images.githubusercontent.com/21338071/58786415-cb85e480-85e7-11e9-9f86-5367fd57d0a0.png)

so either an explicit dependency section like this is needed,

![image](https://user-images.githubusercontent.com/21338071/58786602-3c2d0100-85e8-11e9-91c5-3bb7bd813aa1.png)

or we change the type to project (which i like better). Expect an updated package soon.

**Comment by cgravill** [comment]
Yes, precisely, that shows a lack of dependency on the existing NuGet.

**Comment by kMutagene** [comment]
The new package is up. Seems like i have to be more explicit about the version? Maybe it is enough, can you try it? 

![image](https://user-images.githubusercontent.com/21338071/58807519-84fcae00-8618-11e9-9d81-e56e1c55ef79.png)

**Comment by PrashantVaidyanathan** [comment]
I tried this. I think it might be important to specify the version. 
When I installed it, by default it chose Newtonsoft.Json version 3.5.8 (and also showed a warning stating that the package may not be compatible). I also ran into the same issue (during runtime) as Colin did. 
However, I was able to circumvent it by installing Newstonsoft separately. 

I am still unable to view the nodes. I've been getting a few runtime errors. I suspect I might be running into an issue similar to #16 

OS and framework information:

OS: Windows 10
.Net core SDK version: 2.2.0

openjournals/joss-reviews#1445
